design/synchronizer/auto-failover.md

```diff
@@ -0,0 +1,92 @@
+# Synchronizer Auto-Failover
+
+The following is an overview of the workflow for a Vault Sychronizer to gain leadership:
+
+![](./activity-diagram.png)
+
+Given the following Policy:
+```yml
+- !policy
+  id: conjur
+  body:
+  - !policy
+    id: vault-synchronizer
+    body:
+    - !variable leader-expiration
+
+    # The Group all Synchronizer hosts need to be added to.
+    - !group members
+
+    - !permit
+        role: !group members
+        privileges: [ read, execute, update ]
+        resources:
+      - !variable leader-expiration
+
+      - !policy
+        id: leader
+        owner: !group members
+
+      - !policy
+        id: vault-lines-of-business
+        owner: !group members
+        body:
```

Comment:
I guess this "vault-lines-of-business" policy should be created without body, and Synchronizer should create variables for each LOBs during synchonization intervals.
It would be great to have examples for that as well

Reply:
Great catch. I updated this in the Conjur Intro example above.